src/sql/connection/connection.py

```diff
@@ -597,6 +597,8 @@ def is_use_backtick_template(self):
             return (
                 "`" in sqlglot.Dialect.get_or_raise(cur_dialect).Tokenizer.IDENTIFIERS
             )
+        # Certain dialects like duckdb, oracle, etc do not contain the
+        # class variable IDENTIFIERS
```

Comment:
this comment is confusing, also, the only change in this file is the comment so I'm unsure why this is needed

Reply:
I added it for clarity because I was trying to understand why the test case for `Tokenizer = None` was added since all the dialects that JupySQL supports have the `Tokenizer` attribute.
I'll remove it.

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/966/commits/842abe8f3e5b31fb07a99cb293e4173010c87f44)

Reply:
we can close this for now but adding comments is good, my comment didn't mean to tell you that you should remove the comment but rather to ask for an explanation